cpp/tensorrt_llm/kernels/groupRmsNormKernels/groupRmsNormKernels.cu

```diff
@@ -654,4 +704,95 @@ void GroupRMSNormKernelLargeBatchLauncher(GroupRMSParams<n> params)
 
 INSTANTIATE_GROUP_RMS_NORM_LARGE_BATCH(2)
 
+int getComputeCapabilityMajor()
+{
+    int device;
+    TLLM_CUDA_CHECK(cudaGetDevice(&device));
+    cudaDeviceProp prop;
+    TLLM_CUDA_CHECK(cudaGetDeviceProperties(&prop, device));
+    return prop.major;
+}
+
+bool prefer_base_kernel(int batch, int base_warps, float scheduling_efficiency_ratio)
+{
+    int sm_major = getComputeCapabilityMajor();
+    for (auto const& [known_model, model] : gpu_models)
+    {
+        if (sm_major == known_model)
+        {
+            float p = model.batch_size * batch + model.base_warps * base_warps
+                + model.scheduling_efficiency_ratio * scheduling_efficiency_ratio + model.intercept;
+            p = 1.0f / (1.0f + std::exp(-p));
+            return p > 0.5f;
+        }
+    }
+    return true;
```

Comment:
This is also the fallback path for SM80 and 120. Can you add an info level log here for sm outside the range of `gpu_models`?

Reply:
Added TLLM_LOG_INFO if no matching heuristics.